fortuna/src/Login.js

```diff
@@ -1,49 +1,108 @@
 //@flow strict
 import * as React from 'react';
+import { Link } from 'react-router-dom';
 import 'bootstrap/dist/css/bootstrap.min.css';
-import ReactDOM from 'react-dom';
-import MarketPlace from './MarketPlace.js';
+import LoginPopup from './LoginPopup.js';
 import './Login.css';
-  
+const PORT = 3000;
+
   //This is the Login Page Display
-  class LoginPageDisplay extends React.Component {
-    handleLoginClick = () => {
-        LoginFunction();
-      };
-    render(){
+  class Login extends React.Component<{||}> {
+    state = {
+      response: '',
+      userName: '',
+      password: '',
+      responseToPost: '',
+    };
+    
+    componentDidMount() {
+      this.callApi()
+        .then(res => this.setState({ response: res.express }))
+        .catch(err => console.log(err));
+    };
+    
+    callApi = async () => {
```

Comment:
This is basically an async function, so you should definitely specify what the return type is (especially since it isn't void).